index.html

```diff
@@ -5,6 +5,10 @@
 		<link rel="stylesheet" href="./CSS/index.css">
 	</head>
 	<body>
+			<script
+			src="https://code.jquery.com/jquery-3.3.1.min.js"
```

Comment:
neat that you used jquery. I personally haven't used it yet (it does not play well with React) but being able to show you have used it and are a bit familiar with it is great to put on a resume.